### DISCUSSION TITLE
What is the point of the /@/ alias?

### DISCUSSION THREAD

**Discussion by hyperknot** [cawa-93/vite-electron-builder#1008] [Q&A]
I see that the alias is used in 2 configs, but what is the point? I've commented it out and changed my import to simply load from './' and it works. Both watch and build works. Am I missing something?

**Comment by cawa-93** [comment]
No, you don't miss anything. It was a simple synonym for the root of the package. It was added for convenience, as it is a popular practice. I removed it in the latest update.
